pcap_to_node_pcap/pcap_to_node_pcap.py

```diff
@@ -100,6 +105,15 @@ def get_path(paths):
         print("No path provided: {0}, quitting".format(str(e)))
     return path
 
+def run_split(clients_dir, servers_dir):
+    for tool_cmd in (
+            " ".join(("./PcapSplitter -f", path, "-o", clients_dir, "-m client-ip")),
```

Comment:
where does this function get `path`?

Reply:
It's global due to being defined in __main__, but it should be an arg and not rely on that - fixed.